3ds Max/Max2Babylon/Forms/ExporterForm.cs

```diff
@@ -295,11 +314,13 @@ private async Task<bool> DoExport(ExportItem exportItem, bool multiExport = fals
                     exportNode = exportItem != null ? exportItem.Node : null,
                     pbrNoLight = chkNoAutoLight.Checked,
                     pbrFull = chkFullPBR.Checked,
-                    pbrEnvironment = txtEnvironmentName.Text
+                    pbrEnvironment = txtEnvironmentName.Text,
+                    flattenExport = chkFlatten.Checked
```

Comment:
for our exportParameter, can we use flattenScene

Reply:
done